uber/templates/preregistration/dealer_confirmation.html

```diff
@@ -5,12 +5,12 @@
 <div class="masthead"></div>
 <div class="panel panel-default">
   <div class="panel-body">
-    <h2>{% if c.DEALER_REG_SOFT_CLOSED %}You've been added to our waitlist!{% else %}Thanks for applying as a Dealer!{% endif %}</h2>
+    <h2>{% if group.status == c.WAITLISTED %}You've been added to our waitlist!{% else %}Thanks for applying as a Dealer!{% endif %}</h2>
```

Comment:
This almost certainly hit one dealer last year: the 500th!

Checking for the current status of the system (which is _affected by_ the group existing) rather than the group's status was just silly.